docs/content/en/_index.md

```diff
@@ -12,46 +12,51 @@ description: "K8ssandra documentation: architecture, configuration, guided tasks
 
-## What is K8ssandra?
+* API services (Stargate)
+* Management services (cass-operator)
+* Data backup and restore services (Medusa)
+* Data repair services (Reaper)
+* ServiceMonitor custom resources complete with metric labeling
+* Metrics Collector for Apache Cassandra
+* Management API for Apache Cassandra
 
-K8ssandra is a cloud native distribution of Apache Cassandra® (Cassandra) meant to run on Kubernetes. Accompanying Cassandra is a suite of tools to ease and automate operational tasks. This includes metrics, data anti-entropy services, and backup tooling. As part of K8ssandra's installation process all of these components are installed and wired together freeing your teams from having to perform the tedious plumbing of components.
+And while the initial K8ssandra project features (currently, v1.4.1) are still available, we encourage you to check out K8ssandra Operator. 
 
-Cassandra may be deployed in a number of environments. This includes on bare metal hosts, virtual machines, and within container platforms. Each deployment type has its pros and cons, but in all cases it is **_essential_** that automation be leveraged to ensure that all node are configured homogeneously and without failure.
+## How is this documentation site organized?
+
+We've organized this docs site into three areas:
 
-K8ssandra focuses on deploying Cassandra within Kubernetes. Kubernetes was chosen as it allows for the consumption of a common, versioned, set of APIs and tooling across multiple cloud platforms and environments.
+* [docs.k8ssandra.io](https://docs-staging.k8ssandra.io) provides topics that are of interest to users of K8ssandra Operator and K8ssandra, such as FAQs, Release Notes, and a Glossary.
+* [docs-v1.k8ssandra.io](https://docs-staging-v1.k8ssandra.io) provides topics that are specific to K8ssandra 1.4.x users (the initial project releases).
+* [docs-v2.k8ssandra.io](https://docs-staging-v2.k8ssandra.io) provides topics that are specific to the more recent (and recommended) K8ssandra Operator software.
 
-K8ssandra operator is a v2 feature that allows you to deploy Cassandra into multi-cluster and multi-region Kubernetes environments.
+## What is K8ssandra Operator?
 
-## Why do I want K8ssandra?
+K8ssandra Operator is a cloud native distribution of Cassandra that is designed to run on Kubernetes. Accompanying Cassandra is a suite of tools to ease and automate operational tasks. This includes metrics, data anti-entropy services, and backup tooling. As part of the K8ssandraCluster installation process, which uses the new K8ssandra Operator, all of these components are installed and wired together. This unified operator frees your teams from having to perform the tedious Kubernetes plumbing of components.
```

Comment:
```suggestion
K8ssandra Operator is a native Kubernetes operator that supports the deployment and operation of Cassandra that is designed to run on Kubernetes. Accompanying Cassandra is a suite of tools to ease and automate operational tasks. This includes metrics, data anti-entropy services, and backup tooling. As part of the K8ssandraCluster installation process, which uses the new K8ssandra Operator, all of these components are installed and wired together. This unified operator frees your teams from having to perform the tedious Kubernetes plumbing of components.
```

Reply:
A slight change of wording from what "K8ssandra" was I think is necessary here if we want to say what "K8ssandra Operator" is and not the project as a whole.